ui/app/components/dashboard/quick-actions-card.js

```diff
@@ -30,7 +31,7 @@ export default class DashboardQuickActionsCard extends Component {
 
   get actionOptions() {
     switch (this.selectedEngine.type) {
-      case `kv version ${this.selectedEngine?.version}`:
+      case KV:
```

Comment:
I'm not sure I follow why this is a variable and not a string like the other engines? A string seems sensible to me 🤔

Reply:
I wasn't sure if we do const's for all string types, but seems like we can just do 'kv' directly